src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="form"
+     action="/"
+     method="get"
+     data-qa="big">
```

Comment:
```suggestion
    <form 
      class="form"
      action="/"
      method="get"
      data-qa="big"
   >
```
please fix code style everywhere